.github/workflows/php-compatibility.yml

```diff
@@ -97,7 +97,11 @@ jobs:
         run: phpcs -i
 
       - name: Run PHP compatibility tests
-        run: phpcs --standard=phpcompat.xml.dist -q --report=checkstyle | cs2pr
+        continue-on-error: true
+        run: phpcs --standard=phpcompat.xml.dist --report-full --report-checkstyle=./.cache/phpcs-report.xml
```

Comment:
Should we use a different name for this cache file just in case to avoid any confusion? Maybe `phpcompat-report.xml`?

Reply:
Happy to update, but those files only exist within the runtime of the workflow and are gone as soon as the container has been closed.

Reply:
Updated - all three PHPCS runs now use an unique name for the (temporary) report.